core/src/com/unciv/models/ruleset/unit/BaseUnit.kt

```diff
@@ -597,21 +597,28 @@ class BaseUnit : RulesetObject(), INonPerpetualConstruction {
 
             when {
                 unique.isOfType(UniqueType.Strength) && unique.params[0].toInt() > 0 -> {
-                    if (unique.conditionals.any { it.isOfType(UniqueType.ConditionalVsCity) })
-                        power += (power * unique.params[0].toInt()) / 200
-                    else if (unique.conditionals.any { it.isOfType(UniqueType.ConditionalVsUnits) })
-                        power += (power * unique.params[0].toInt()) / 400
+                    if (unique.conditionals.any { it.isOfType(UniqueType.ConditionalVsUnits) } ) { // Bonus vs some units - a quarter of the bonus
```

Comment:
Should convert to int, divide by 4, and only them convert to percent

This will give you, for a 4% bonus that should become a 1%:
(1.04)/4=0.26 instead of 1.01